src/modules/mc_hover_thrust_estimator/hover_thrust_estimator_params.c

```diff
@@ -77,7 +77,7 @@ PARAM_DEFINE_FLOAT(HTE_ACC_GATE, 3.0);
  * @decimal 3
  * @min 0.0
  * @max 1.0
- * @unit normalized_thrust
+ * @unit %alized_thrust
```

Comment:
@bkueng I think this one can just be "%" right? The fact that it is normalised thrust is implied by the description.

Reply:
@hamishwillee Yes, we can use % but then we need to scale the result `0-100% -> 0.0-1.0`. Is that automatically done in QGC?

Reply:
@bresch The simple answer is that I don't know. For now I have reverted this and we can revisit in a later round.

@bkueng This affects everything else we changed from "norm". I guess we should hold off merging until we know (unless you know?)

Reply:
Scaling is done on the UI side, so no other change required.

Reply:
![Screen Capture_select-area_20200814100721](https://user-images.githubusercontent.com/14822839/90228133-fdaeeb00-de15-11ea-8a6f-f4c0a38875fc.png)
Hmm, it seems that the `@unit norm` is automatically converted and displayed as `%` in QGC